3-projects/README.md

```diff
@@ -10,25 +10,26 @@ The purpose of this step is to setup folder structure and projects for applicati
 
 ## Usage
 ### Setup to run via Cloud Build
-1. Clone repo gcloud source repos clone gcp-projects --project=YOUR_CLOUD_BUILD_PROJECT_ID
-1. Change freshly cloned repo and change to non master branch git checkout -b plan
-1. Copy contents of foundation to new repo cp ../terraform-example-foundation/3-projects/* .
+1. Clone repo `gcloud source repos clone gcp-projects --project=YOUR_CLOUD_BUILD_PROJECT_ID`
+1. Change freshly cloned repo and change to non master branch `git checkout -b plan`
+1. Copy contents of foundation to new repo `cp ../terraform-example-foundation/3-projects/* .` (modify accordingly based on your current directory)
```

Comment:
I guess it's cp -R ../terraform-example-foundation/3-projects/* . here to include the module folder